habitat-baselines/habitat_baselines/rl/hrl/hierarchical_policy.py

```diff
@@ -426,8 +426,20 @@ def from_config(
         orig_action_space,
         **kwargs,
     ):
+        agent_name = None
```

Comment:
This code is the same as the one in resnet_policy. 
I think it will be easier to reuse later if you made a helper method that does just that.